backend/test_util.go

```diff
@@ -0,0 +1,20 @@
+package backend
+
+import (
+	"bytes"
+	"compress/gzip"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// Gz returns a compressed version of its input string
+func Gz(t *testing.T, s string) string {
```

Comment:
Test utilities go in `fstest/fstest.go`

However I think this probably doesn't belong in this commit.

Reply:
This was related to the new B2 tests. I've backed it out of this PR.